cni/network/network_windows.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/Azure/azure-container-networking/network"
 	"github.com/Azure/azure-container-networking/network/policy"
```

Comment:
Also share what all testing has been performed? Just containerd or docker has been tested to make HnsV1 is working as earlier too.

Reply:
Please confirm?

Reply:
Specifically the portmapping scenario